x/warp/module.go

```diff
@@ -48,9 +48,8 @@ func (*configurator) MakeConfig() precompileconfig.Config {
 
 // Configure is a no-op for warp since it does not need to store any information in the state
 func (*configurator) Configure(chainConfig precompileconfig.ChainConfig, cfg precompileconfig.Config, state contract.StateDB, _ contract.ConfigurationBlockContext) error {
-	config, ok := cfg.(*Config)
-	if !ok {
-		return fmt.Errorf("incorrect config %T: %v", config, config)
+	if _, ok := cfg.(*Config); !ok {
+		return fmt.Errorf("expected config type %T, got %T: %v", &Config{}, cfg, cfg)
```

Comment:
We do have same issue in other precompiles + template. Could you do a search + replace to fix others as well?

Reply:
Done.